package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "mesto",
+  "version": "0.1.1",
+  "description": "A training project, a photo gallery site that allows you to add, delete and like cards with photos of beautiful places in Russia",
+  "main": "index.js",
+  "scripts": {
+    "build": "NODE_ENV=production rimraf dist && webpack --mode production",
```

Это код из практики, но здесь ошибка. 
`NODE_ENV` передается в команду `rimraf`, но не в `webpack`. Это приводит к тому, что не работает вот эта строчка:
https://github.com/ivan-markin/mesto/blob/37eafee6f4f3ef2281a00af314a315e7e8d88a51/webpack.config.js#L28
Потому что `isDev` будет равен `false`.

```suggestion
    "build": "rimraf dist && NODE_ENV=production webpack --mode production",
```

---

Исправил